base_local_planner/src/speed_limiter.cpp

```diff
@@ -198,23 +212,18 @@ void SpeedCostFunction::calculateFootprintBounds(std::vector<geometry_msgs::Poin
     footprint_min_y_ = std::numeric_limits<double>::max();
     footprint_min_x_ = std::numeric_limits<double>::max();
 
+    circumscribed_radius_ = std::numeric_limits<double>::max();
+
     for (auto pt : footprint)
     {
       if (pt.x < footprint_min_x_) {footprint_min_x_ = pt.x;}
       if (pt.x > footprint_max_x_) {footprint_max_x_ = pt.x;}
       if (pt.y < footprint_min_y_) {footprint_min_y_ = pt.y;}
       if (pt.y > footprint_max_y_) {footprint_max_y_ = pt.y;}
+      double dist = std::sqrt(pt.x * pt.x + pt.y * pt.y);
+      if (dist < circumscribed_radius_) {circumscribed_radius_ = dist;}
```

Comment:
Isn't this finding the inscribed radius?

Reply:
Good catch.  I'm not even quite finding the inscribed radius, just some number in between.  Updating it in the next commit.